roles/unbind-keycloak-apb/tasks/main.yml

```diff
@@ -23,10 +23,10 @@
 -
   name: delete installation details bearer
```

Comment:
We should probably update the name here to `Deleting {{ _apb_bind_creds.resource }} client` or something like that to be more descriptive.